src/tools/ceph_objectstore_tool.cc

```diff
@@ -2338,7 +2341,8 @@ int main(int argc, char **argv)
     usage(desc);
     myexit(1);
   }
-  if (op != "list" && op != "fsck" && vm.count("op") && vm.count("object")) {
+  if (op != "list" && op != "fsck" && op != "fuse" &&
```

Comment:
Checking for --op "fuse" (or "fsck" for that matter) are not needed here, unless they expect positional arguments.